app/routes/overzicht.js

```diff
@@ -0,0 +1,16 @@
+import Route from '@ember/routing/route';
+
+import { service } from '@ember/service';
+
+export default class OverzichtRoute extends Route {
+  @service currentSession;
+  @service session;
+  @service router;
+
+  beforeModel(transition) {
+    this.session.requireAuthentication(transition, 'login');
+    if (this.currentSession.isAdmin) {
+      this.router.replaceWith('overzicht.admin');
```

Comment:
odd. i would just have kept one route and would have made it conditional with {{#if }} statements, no?

Reply:
Isn't the index component not getting to big in that case?